### PR TITLE
docs: fix typo on kitex/Tutorials/basic_feature/ciruitbreaker

### PR DESCRIPTION
**Pull request by wpc0086**
1、熔断器使用说明有误,官网说默认提供的是 circuitbreaker.RPCInfo2Key，但是实际上是circuitbreak.RPCInfo2Key,并没有er。 2、修改了使用实例，更直观！

#### What type of PR is this?

docs: Documentation only changes

#### What this PR does / why we need it (en: English/zh: Chinese):

en: Modify the wrong place in the instructions of the fuse tutorial. The default is circuitbreaker.RPCInfo2Key instead of circuitbreaker.RPCInfo2Key. The usage example has been modified to be more intuitive!
zh:修改熔断器教程中使用说明中错误的地方,默认提供的是 circuitbreak.RPCInfo2Key而不是默认提供的是 circuitbreaker.RPCInfo2Key，修改了使用示例，更直观！

#### Which issue(s) this PR fixes:

Fixes #<538>

### CODE REVIEW

**Comment by GuangmingLuo** [comment]
> [CLA assistant check](https://cla-assistant.io/cloudwego/cloudwego.github.io?pullRequest=539) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/cloudwego/cloudwego.github.io?pullRequest=539) before we can accept your contribution.
> 
> **王鹏程** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/cloudwego/cloudwego.github.io?pullRequest=539) it.

Please sign this  CLA first.

**Review by jayantxie — APPROVED** [review]

File: content/en/docs/kitex/Tutorials/basic-feature/circuitbreaker.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The following document will introduce that how to enable circuit breaker and con
 
 ```go
 // build a new CBSuite
-cbs := circuitbreak.NewCBSuite(GenServiceCBKeyFunc)
+cbs := circuitbreak.NewCBSuite(circuitbreak.RPCInfo2Key)//GenServiceCBKeyFunc
```
Comment (jayantxie) [inline content/en/docs/kitex/Tutorials/basic-feature/circuitbreaker.md:19]: 这里保持原样吧，避免误以为只能使用circuitbreak.RPCInfo2Key

**Comment by GuangmingLuo** [comment]
@wpc0086 hello, any progress ?

File: content/en/docs/kitex/Tutorials/basic-feature/circuitbreaker.md (hunk 1 above)
Comment (wpc0086) [reply content/en/docs/kitex/Tutorials/basic-feature/circuitbreaker.md]: 好的，使用实例已经恢复默认了

**Comment by wpc0086** [comment]
> @wpc0086 hello, any progress ?
抱歉，已经按照jayantxie的建议修改了

**Review by GuangmingLuo — APPROVED** [review]

**Pull request merged**